test/game_test.rb

```diff
@@ -0,0 +1,27 @@
+require 'simplecov'
+SimpleCov.start
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/game'
+
+class GameTest < Minitest::Test
+
+  def test_it_exists
+    game = Game.new({:game_id => "2012030221", :season => "20122013", :type => "Postseason", :date_time => "5/16/13", :away_team_id => 3, :home_team_id => 6, :away_goals => 2, :home_goals => 3, :venue => "Toyota Stadium"})
```

Comment:
Should the numeric values in this hash be integers to match the actual hashes we're using to create the game objects?